### PR TITLE
#1346 : Remove outdated frameworks from examples/frameworks/

### PR DESCRIPTION
**Pull request by vickysam**
Issue : Remove outdated frameworks from examples/frameworks/ #1346

### CODE REVIEW

File: examples/frameworks/cherryapp.py [hunk 1]
```diff
@@ -3,8 +3,9 @@
 cherrypy.config.update({'environment': 'embedded'})
 
 class Root(object):
+    @cherrypy.expose
     def index(self):
         return 'Hello World!'
     index.exposed = True
```
Comment (berkerpeksag) [inline examples/frameworks/cherryapp.py]: You forgot to remove this line.

File: examples/frameworks/cherryapp.py [hunk 2]
```diff
@@ -3,8 +3,9 @@
...
 
-app = cherrypy.Application(Root(), script_name=None, config=None)
+app = cherrypy.Application(Root(), script_name=None, config=None)
```
Comment (berkerpeksag) [inline examples/frameworks/cherryapp.py]: We can probably use `app = cherrypy.tree.mount(Root())` here. Can you please try with the latest CherryPy?

Comment (berkerpeksag) [inline examples/frameworks/cherryapp.py]: Also, please fix the "No newline at end of line" warning.

File: examples/frameworks/cherryapp.py [hunk 3]
```diff
@@ -3,8 +3,12 @@
 cherrypy.config.update({'environment': 'embedded'})
 
 class Root(object):
+    @cherrypy.expose
```
Comment (vickysam) [inline examples/frameworks/cherryapp.py]: New annotation in cherrypi, so kept it

File: examples/frameworks/cherryapp.py (hunk 2 above)
Comment (vickysam) [inline examples/frameworks/cherryapp.py]: Updated with new cherrypy

File: examples/frameworks/cherryapp.py [hunk 4]
```diff
@@ -3,8 +3,12 @@
...
     def index(self):
         return 'Hello World!'
-    index.exposed = True
 
-app = cherrypy.Application(Root(), script_name=None, config=None)
+cherrypy.config.update({'engine.autoreload.on': False})
```
Comment (berkerpeksag) [inline examples/frameworks/cherryapp.py]: According to http://docs.cherrypy.org/en/latest/deploy.html#embedding-into-another-wsgi-framework, with `cherrypy.config.update({'environment': 'embedded'})` this line is not needed.

Could you also move line 3 here? Like

```py
cherrypy.config.update({'environment': 'embedded'})
cherrypy.server.unsubscribe()
cherrypy.engine.start()
app = cherrypy.tree.mount(Root())
```

File: examples/frameworks/cherryapp.py [hunk 5]
```diff
@@ -3,8 +3,12 @@
...
+cherrypy.server.unsubscribe()
+cherrypy.engine.start()
+app = cherrypy.tree.mount(Root())
+
```
Comment (berkerpeksag) [inline examples/frameworks/cherryapp.py]: This line can be removed.

Comment (vickysam) [inline examples/frameworks/cherryapp.py]: should i remove 
`app = cherrypy.tree.mount(Root())
this line also?`

Comment (vickysam) [inline examples/frameworks/cherryapp.py]: should i remove all these lines?

File: examples/frameworks/cherryapp.py (hunk 4 above)
Comment (vickysam) [inline examples/frameworks/cherryapp.py]: which line 3?

Comment (berkerpeksag) [inline examples/frameworks/cherryapp.py]: Line 3 in cherryapp.py:

```py
cherrypy.config.update({'environment': 'embedded'})
```

File: examples/frameworks/cherryapp.py (hunk 5 above)
Comment (berkerpeksag) [inline examples/frameworks/cherryapp.py]: No, I meant the empty line.

**Comment by berkerpeksag** [comment]
Merged in https://github.com/benoitc/gunicorn/commit/e549835c544bbcce9aa62b7afed39f0c6d889cfb.

**Comment by vickysam** [comment]
Thanks berkerpeksag :)

**Pull request closed without merging**